src/App.js

```diff
@@ -1,12 +1,80 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import PokemonTile from './assets/component/pokemon-tile/PokemonTile';
+import axios from "axios";
 import './App.css';
 
+
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [resultData, setResultData] = useState('');
```

Comment:
Ik zou de default waarde `undefined` maken, omdat het type van deze waarde nooit string zal zijn
```suggestion
    const [resultData, setResultData] = useState();
```